test/Microsoft.NET.Pack.Tests/GivenThatWeWantToPackASimpleLibrary.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using static Microsoft.NET.TestFramework.Commands.MSBuildTest;
+
+namespace Microsoft.NET.Pack.Tests
+{
+    public class GivenThatWeWantToPackASimpleLibrary
```

Comment:
Is this test related to the OutputPath changes or is it just filling in more general test coverage?